src/PkgCacheInspector.jl

```diff
@@ -18,29 +18,31 @@ struct PkgCacheSizes
     fptrlist::Int
 end
 
+const cache_displaynames = [
+        "system",
+        "isbits",
+        "symbols",
+        "tags",
+        "relocations",
+        "gvars",
+        "fptrs"
+    ]
+const cache_displaynames_l = maximum(length, cache_displaynames)
+
 function Base.show(io::IO, szs::PkgCacheSizes)
     indent = get(io, :indent, 0)
     nd = ntot = 0
     nf = length(fieldnames(typeof(szs)))
+    l = cache_displaynames_l
     for i = 1:nf
         nb = getfield(szs, i)
         nd = max(nd, ndigits(nb))
         ntot += nb
     end
-    displaynames = [
-        "system",
-        "isbits",
-        "symbols",
-        "tags",
-        "relocations",
-        "gvars",
-        "fptrs"
-    ]
-    l = maximum(length, displaynames)
     println(io, " "^indent, "Segment sizes (bytes):")
     for i = 1:nf
         nb = getfield(szs, i)
-        println(io, " "^indent, rpad(displaynames[i] * ": ", l+2), lpad(string(nb), nd), " (", @sprintf("% 6.2f", 100*nb/ntot), "%)")
+        println(io, " "^indent, rpad(cache_displaynames[i] * ": ", l+2), lpad(string(nb), nd), " (", @sprintf("% 6.2f", 100*nb/ntot), "%)")
```

Comment:
```suggestion
        println(io, " "^indent, rpad(cache_displaynames[i] * ": ", cache_displaynames_l+2), lpad(string(nb), nd), " (", @sprintf("% 6.2f", 100*nb/ntot), "%)")
```
and then delete the assignment of `l` above.